include/torch-mlir/Dialect/Torch/IR/TorchTypes.h

```diff
@@ -35,7 +35,7 @@ using GetTensorTypeFn =
     llvm::function_ref<Type(MLIRContext *, Optional<ArrayRef<int64_t>>, Type)>;
 
 /// The representation of an unknown dimension size in an ArrayRef<int64_t>.
-constexpr static int64_t kUnknownSize = -1;
+constexpr static int64_t kUnknownSize = std::numeric_limits<int64_t>::min();
```

Comment:
Nit: it seems better to use `ShapedType::kDynamicSize` (from "mlir/IR/BuiltinTypes.h") here, so that we don't have to do another refactoring if the value changes in the future.